_posts/2022-03-21-introducing-open-source-places.md

```diff
@@ -0,0 +1,97 @@
+
+### How can it be used?
+{: .text-left}
+
+We think there will be numerous potential use cases for such a large and varied dataset, for state and local agencies, citizens, and even researchers. The data could be used to perform different analyses on certain business types or even specific businesses. How many Beans and Brews locations are within a half-mile of Starbucks? The businesses and landmarks could be used as a reference for 911 call-takers, first responders, or emergency managers. This can be particularly helpful because we don't always know the address of our emergencies, but we usually know a point of interest or place name. The data could even be used to quality control other datasets by looking for features that we may be missing. Are we missing any schools, fire stations, liquor stores, or cemeteries from our authoritative datasets?
+
+### How can I participate?
+{: .text-left}
+
+It's pretty easy, really. All you need to do is create a free [OSM account](https://www.openstreetmap.org/user/new), login, start editing or creating places (iD is the default editor), and save your updates. Then, the next time we run our python script to update this layer, your changes will be included and published in the SGID. In addition to the default, web-based editor on openstreetmap.org ([iD](https://www.openstreetmap.org/edit?editor=id)), there are several desktop and mobile (Android and iOS) editors that can be used, including, [JOSM](https://josm.openstreetmap.de/), [Vespucci](https://vespucci.io/), and [GoMap!!](https://apps.apple.com/app/id592990211). The [ArcGIS OpenStreetMap Editor](https://github.com/Esri/arcgis-osm-editor) (ArcGIS Desktop 10+ only) is also available for ESRI users. A larger list of editors is maintained on the [OSM Wiki](https://wiki.openstreetmap.org/wiki/Comparison_of_editors). If creating an OSM account and using an editor are too cumbersome, you can also submit businesses anonymously with a simple form entry and pin-location at [OnOSM](http://onosm.org). Your submissions through OnOSM will be reviewed and added by an OSM user.
+
+You can get involved in the OSM community by checking out upcoming events on the [OSM Calendar](https://osmcal.org/?in=United%20States).  The calendar will list events across the US, including those right here in Utah.  The [OSM Slack community](https://slack.openstreetmap.us/) is also open to anyone, with a specific `#local-utah` channel for users and events in the state.
+
+![Epic Handshake]({% link images/os_handshake.png %} "Epic Handshake"){: .flex .flex--center}
+
+### How does it differ from other SGID data?
+{: .text-left}
+
+- It's **not** authoritative - due to the nature of crowdsourced data, we can't guarantee that it's perfect or accurate
+- It has a different license - instead of our usual [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) license, it carries the [ODbL 1.0](https://opendatacommons.org/licenses/odbl/summary/) license
```

Comment:
**suggestion**: link to our license page 

```suggestion
- It has a different license - instead of our usual [CC BY 4.0]({% link about/policy/license-disclaimer/index.md %}}) license, it carries the [ODbL 1.0](https://opendatacommons.org/licenses/odbl/summary/) license
```